openshift/template.yaml

```diff
@@ -0,0 +1,173 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: f8a-gemini-server
```

Comment:
@GeetikaBatra let's make `fabric8-gemini-server` consistent across the template. Replace all the references of `f8a-gemini-server` with `fabric8-gemini-server`